### PR TITLE
SAK-44905 Clean up name and description form field styling

### PR DESCRIPTION
**Pull request by davidpbauer**
Improve the styling of the name and description form fields in the
edit dialog and on the checklist page by adding spacing and making
them wider.

### CODE REVIEW

File: lessonbuilder/tool/src/webapp/templates/Checklist.html [hunk 1]
```diff
@@ -51,7 +51,7 @@
             <p class="checklistFormInput">
                 <label class="checklistDescription" rsf:id="msg=simplepage.new-checklist-description" for="description"></label>
                 <br />
-                <textarea rsf:id="description" id="description"></textarea>
+                <textarea rsf:id="description" id="description" rows="6"></textarea>
```
[the remark this answers is not in the document]
Comment (davidpbauer) [reply lessonbuilder/tool/src/webapp/templates/Checklist.html]: Added aria-labelledby

File: lessonbuilder/tool/src/webapp/templates/Checklist.html [hunk 2]
```diff
@@ -49,9 +49,9 @@
                 <input type="checkbox" rsf:id="isNameHidden" id="isNameHidden" class="hideNameCheckbox"/><label rsf:id="nameHidden-label" for="isNameHidden" class="hideNameCheckboxLabel"></label>
             </p>
             <p class="checklistFormInput">
-                <label class="checklistDescription" rsf:id="msg=simplepage.new-checklist-description" for="description"></label>
+                <label class="checklistDescription" rsf:id="msg=simplepage.new-checklist-description" id="checklistDescriptionLabel" for="description"></label>
                 <br />
-                <textarea rsf:id="description" id="description"></textarea>
+                <textarea rsf:id="description" id="description" aria-labelledby="checklistDescriptionLabel" rows="6"></textarea>
```
[the remark this answers is not in the document]
Comment (davidpbauer) [reply lessonbuilder/tool/src/webapp/templates/Checklist.html:54]: I was under the impression that aria-labelledby was a perfectly acceptable to label a form element with text from another element in the DOM... 

Is this not the right way to address this?

[the remark this answers is not in the document]
Comment (ottenhoff) [reply lessonbuilder/tool/src/webapp/templates/Checklist.html:54]: i think aria-labelledby is unnecessary here. the label with the for attribute seems like all you need, right?

`<label for="txtboxMultiline">Enter the tags for the article</label>
<textarea id="txtboxMultiline" required></textarea>`

[the remark this answers is not in the document]
Comment (davidpbauer) [reply lessonbuilder/tool/src/webapp/templates/Checklist.html:54]: That was my initial thought as well. The bot flagged it the first time so I added the aria-labelledby attribute to try to make it happy.

I can take it back out.

**Review by ottenhoff — APPROVED** [review]

**Pull request merged**